### DISCUSSION TITLE
statdata v 2.1.1 problem with saving the entire URL

### DISCUSSION THREAD

**Discussion by warehouser** [PrestaShop/PrestaShop#36469] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

It's about storing the input URL in the database
The URL is stored in the ps_connections_source array, request_uri column.
Did you notice that this column only holds 255 characters?

I changed the number of characters for this column but it still saves a truncated URL of 255 characters. Is it possible to correct this in the code?

Many modules that integrate with Google Analytics use this data. For example, it retrieves the gclid parameters that are truncated most often.

I report the need to increase the URL storage length to 1000 characters.

### Expected behavior

Fix to 1000 characters.

### Steps to reproduce

1. Go to phpmyadmin
2. Table ps_connections_source
3. column request_uri column

### PrestaShop version(s) where the bug happened

1.7.8.8

### PHP version(s) where the bug happened

7.4

### Your company or customer's name goes here (if applicable).

statsdata v 2.1.1

**Comment by florine2623** [comment]
Moved to Discussions, because this is an improvement idea
